app/views/layouts/application.html.erb

```diff
@@ -1,47 +1,42 @@
 <% content_for :body do %>
-  <% if use_govuk_account_layout? %>
-    <div id="wrapper">
-      <% content_for :before_content do %>
-        <%= yield :back_link %>
-      <% end %>
-      <%= yield :before_content %>
-      <main role="main" id="content">
-        <%= yield %>
-      </main>
-    </div>
-  <% else %>
-    <div class="govuk-width-container" id="wrapper">
-      <%= yield :back_link %>
-      <main class="govuk-main-wrapper<%= " govuk-main-wrapper--l" if yield(:back_link).blank?%>" role="main" id="content">
-        <div id="email-alert-frontend">
-          <div class="govuk-grid-row">
-            <div class="govuk-grid-column-two-thirds">
-              <%= yield %>
-            </div>
+  <div class="govuk-width-container" id="wrapper">
+    <%= yield :back_link %>
+    <main class="govuk-main-wrapper<%= " govuk-main-wrapper--l" if yield(:back_link).blank?%>" role="main" id="content">
+      <div id="email-alert-frontend">
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
+            <%= yield %>
           </div>
         </div>
-      </main>
-    </div>
-  <% end %>
+      </div>
+    </main>
+  </div>
 <% end %>
 
-<!DOCTYPE html>
-<html>
-  <head>
-    <title><%= yield :title %> - GOV.UK</title>
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-    <% if content_for(:meta_description).present? %>
-      <meta name="description" content="<%= content_for(:meta_description) %>" />
-    <% end %>
-    <%= yield :head %>
-    <meta name="robots" content="noindex, nofollow">
-    <%=
-      render_component_stylesheets
-    %>
-  </head>
-  <body class="govuk-template__body">
-    <%= yield :body %>
-    <%= javascript_include_tag 'application', type: "module" %>
-  </body>
-</html>
+<%= content_for :head do %>
+  <%= csp_meta_tag %>
+  <meta name="robots" content="noindex, nofollow">
+
+  <% add_gem_component_stylesheet("cookie-banner") %>
```

Comment:
Just to check - these are all the stylesheets that are needed prior to something loading, which is why they need to be hoisted here, and not included in application.scss? Is that right?

Reply:
application.scss should include stylesheets needed by all pages, but these stylesheets are ones only needed by these specific layouts, so they're hoisted above the call to layout_for_public (in fact there's now nothing in this specific layout, but the account layout has the cross-service-header and phase-banner stylesheets included, since they're needed for anything using that layout, but not for anything using the normal layout).